samples/ChangeFeedProcessor/DocumentDB.ChangeFeedProcessor/ChangeFeedEventHost.cs

```diff
@@ -81,7 +81,7 @@ namespace DocumentDB.ChangeFeedProcessor
     /// </example>
     public class ChangeFeedEventHost : IPartitionObserver<DocumentServiceLease>
     {
-        const string DefaultUserAgentSuffix = "changefeed-0.2";
+        const string DefaultUserAgentSuffix = "changefeed-0.3";
```

Comment:
If this will be packed in a Nuget eventually, might want to put the next nuget version (right now it's on 1.1.1, could be 1.2?).

Reply:
Since this might be brought into the private repos for publishing 😄

Reply:
We can do that when we sync with nuget version. For now probably make sense to keep as 0.*